.gitignore

```diff
@@ -71,7 +71,7 @@ environment.sh
 .envrc
 .env
 
-celerybeat-schedule
+celerybeat-schedule.db
```

Comment:
this is the schedule file my machine was making 🤷‍♂️

Reply:
Omar will bring in new changes [in this PR](https://github.com/cds-snc/notification-api/pull/1321/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R75) to include all files, so I would remove the one change in your PR (or make it the same as his).